### PR TITLE
added message to display when the request fails

### PR DESCRIPTION
**Pull request by shikhin-metron**

### CODE REVIEW

**Comment by shikhin-metron** [comment]
Screenshot of message
Screenshot from 2022-03-23 17-09-24

**Comment by devang-metron** [comment]
@kapil-metron Please check the error and check if anything else needs to be added.

File: dsapi/service/ds_base_service.py [hunk 1]
```diff
@@ -46,14 +46,16 @@ def _request(self, path, method='GET', body=None, headers=None):
         """
         url = '{}{}'.format(self._url_base, path)
         headers = self._headers() if headers is None else headers
+        headers["Accept"] = "*/*"
         response, content = super(DSBaseService, self)._request(url,
                                                                 method=method,
                                                                 body=str(body).replace("'", '"'),
                                                                 headers=headers)
         if int(response['status']) == 200:
```
Comment (kapil-metron) [inline dsapi/service/ds_base_service.py]: Some of the DS API return 204 as well, so I think we should consider 200 to 299 as success.

**Review by kapil-metron — CHANGES_REQUESTED** [review]

File: dsapi/service/ds_base_service.py (hunk 1 above)
Comment (devang-metron) [reply dsapi/service/ds_base_service.py]: @shikhin-metron Please check if the api calls made with this function are returning which status codes.

Comment (shikhin-metron) [reply dsapi/service/ds_base_service.py]: will add condition to accept for all status codes between 200 and 299. Will that be correct?

**Review by kapil-metron — APPROVED** [review]

**Pull request merged**